library-app/src/components/Books/BooksList/BooksList.tsx

```diff
@@ -1,18 +1,62 @@
 import BooksItem from '../BooksItem/BooksItem'
 import styles from './BooksList.module.css'
+import { useEffect, useState } from 'react'
+import Modal from '../../Layout/Modal'
+import ManageBookForm from './ManageBookForm'
+import { getBooksRequest } from '../../../services/BooksServices'
+import Author from '../../../models/author.model'
+
+export interface BookBodyDataGet {
+  Id: number
+  Title: string
+  Description: string
+  Isbn: string
+  Quantity: number
+  Cover: string
+  PublishDate: string
+  Authors: Author[]
+}
 
 const BooksList = () => {
+  const [isOpen, setIsOpen] = useState(false)
+  const [books, setBooks] = useState<BookBodyDataGet[]>([
+    {
+      Id: 0,
+      Title: '',
+      Description: '',
+      Isbn: '',
+      Quantity: 0,
+      Cover: '',
+      PublishDate: '',
+      Authors: [],
+    },
+  ])
+
+  useEffect(() => {
+    fetchData()
+  }, [])
+
+  const fetchData = async () => {
+    try {
+      const responseData = await getBooksRequest()
+      setBooks(responseData.data)
+      return responseData
+    } catch (error) {
+      console.error(error)
+    }
+  }
+
   return (
     <div className={styles.wrapp}>
-      <BooksItem />
-      <BooksItem />
-      <BooksItem />
-      <BooksItem />
-      <BooksItem />
-      <BooksItem />
-      <BooksItem />
-      <BooksItem />
-      <BooksItem />
+      <div className={styles.background}></div>
+
+      <Modal open={isOpen} onClose={() => setIsOpen(false)}>
+        <ManageBookForm />
+      </Modal>
+      <div className={styles['books-wrap']}>
+        {!books && <p>Nema dostupnih knjiga</p>}
```

Comment:
You can use the ternary operator here; you don't need to duplicate conditions because you have only one if-else.
Something like this: 
`<div className={styles['books-wrap']}>
        {!books ? <p>Nema dostupnih knjiga</p> : {books.map((book) => <BooksItem key={book.Id} book={book} />)}
      </div>`

Reply:
corrected